components/admin/DateInput.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "@styles/admin.module.css";
+
+type Props = {
+  value: string;
+  placeholder: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  name: string;
+  label: string;
+};
+
+export default function Dateinput(props: Props) {
+  const { value, placeholder, onChange, label, name } = props;
+
+  return (
+    <div className="flex flex-col gap-2">
+      <label htmlFor={name}>{label}</label>
+      <input
+        id={name}
+        name={name}
+        value={value}
+        onChange={onChange}
+        placeholder={placeholder}
+        className={styles.dateInput}
+        type="date"
+      />
+    </div>
+  );
+}
```

Comment:
I think we could have reused material UI (or your library) components here. 

Making our owns is not very optimized :/. I guess you followed Kevin on this but you should understand for next time that kevin might not have the context on how long it is to take a customized component here and you should get back to him to ask him if we can use material UI (he knows what it is).

Reply:
issue is - material ui uses datepicker inside their "labs" package. This adds an overhead of handling a new library just to use a date component in an internal dashboard. I took the decision to not make any changes and use which is defined inbuilt by html since it handles all edge cases by default and i only have to use it as it is using the `input` tag.